DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
+|`temporary-failure`|The provider was unable to deliver message, email inbox was full or phone was turned off; you can try to send the message again|Yes|Yes||
+|`technical-failure`|Email / Text: Notify had a technical failure; you can try to send the message again. <br><br>Letter: Notify had an unexpected error while sending to our printing provider. <br><br>You can leave out this argument to ignore this filter.|Yes|Yes||
+|`accepted`|Notify is printing and posting the letter|||Yes|
+|`received`|The provider has received the letter to deliver|||Yes|
 
-##### `template_type`
+#### notificationType (optional)
 
-If omitted all messages are returned. Otherwise you can filter by:
+You can filter by:
 
 * `email`
 * `sms`
 * `letter`
 
-You can omit this argument to ignore the filter.
-
-##### `status`
+#### reference (optional)
 
-You can filter by:
-
-* `sending` - the message is queued to be sent by the provider.
-* `delivered` - the message was successfully delivered.
-* `failed` - this will return all failure statuses `permanent-failure`, `temporary-failure` and `technical-failure`.
-* `permanent-failure` - the provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list.
-* `temporary-failure` - the provider was unable to deliver message, email box was full or the phone was turned off; you can try to send the message again.
-* `technical-failure` - Notify had a technical failure; you can try to send the message again.
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-You can omit this argument to ignore the filter.
+```ruby
+reference: 'your_reference_string';
+```
 
-##### `reference`
+#### older_than (optional)
 
-This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
+Input the ID of a notification into this argument. If you use this argument, the client returns the next 250 received notifications older than the given ID.
 
-You can omit this argument to ignore the filter.
+```ruby
+older_than: 'e194efd1-c34d-49c9-9915-e4267e01e92e'
+```
 
+If you leave out this argument, the client returns the most recent 250 notifications.
 
-##### `older_than`
-You can get the notifications older than a given `Notification.id`.
-You can omit this argument to ignore this filter.
+The client will only return notifications that are 7 days old or less. If the notification specified in this argument is older than 7 days, the client will return an empty `collection` response.
 
-</details>
+### Response
 
+If the request to the client is successful, the client will return a `Notifications::Client::NotificationsCollection`.
 
+You must then run either `notifications.links` or `notifications.collection` on the `Notifications::Client::NotificationsCollection` object:
```

Comment:
I think I would probably describe this as
'You can either call the `notifications.links` method or the `notifications.collection` method ...' Or something along those lines - basically describing it as calling methods instead of running things.